crates/core/src/vm.rs

```diff
@@ -518,15 +533,15 @@ impl<'db, 'tx> DbProgram<'db, 'tx> {
                         }
                     }
                 }
-                Ok(Code::Pass)
+                Ok(Code::Pass(None)) // TODO going to assume dropping a table doesn't need to broadcast its entire contents as a notify
             }
             TxMode::Tx(_) => unreachable!("mutable operation is invalid with read tx"),
         }
     }
 
     fn _set_config(&mut self, name: String, value: AlgebraicValue) -> Result<Code, ErrorVm> {
         self.db.set_config(&name, value)?;
-        Ok(Code::Pass)
+        Ok(Code::Pass(None)) // TODO can subscriptions watch config values?
```

Comment:
Unsure how to broadcast these, as there's no "sql update" happening here. Is there a path to notify config changes?

Reply:
Clients should only care about inserts and deletes in non-system tables; anything else doesn't need to be broadcast.